src/Flowpipes/flowpipes.jl

```diff
@@ -379,6 +379,13 @@ function Convexify(fp::Flowpipe{N, ReachSet{N, ST}}) where {N, ST}
     return ReachSet(Y, tspan(fp))
 end
 
+function Convexify(fp::AbstractVector{SparseReachSet{N, ST, D}}) where {N, ST, D}
```

Comment:
seems like we should dispatch on `AbstractLazyReachSet` and use the existing function.

Reply:
That's much better. 👍